boards/BMS/Inc/App/states/App_ChargeState.h

```diff
@@ -1,9 +1,11 @@
 #pragma once
 
 #include "App_SharedStateMachine.h"
-
+#include "App_Accumulator.h"
 /**
  * Get a pointer to the Charge State.
  * @return A pointer to the Charge State. THIS SHOULD NOT BE MODIFIED.
  */
 const struct State *App_GetChargeState(void);
+// 0.05C is standard for a boundary to consider full charge
+#define CURRENT_AT_MAX_CHARGE (0.05f * C_RATE_TO_AMPS);
```

Comment:
This should be placed above function definitions, also add an empty line to end of file